### PR TITLE
feat: wormhole bridge widget FE integration

### PR DESCRIPTION
**Pull request by ChefBingbong**
---

## PR-Codex overview
This PR adds support for multiple bridge disclaimers and integrates jotai for state management. It also introduces new components and updates dependencies.

### Detailed summary
- Added bridge disclaimers for Stargate, Alexar, and Wormhole
- Integrated jotai for state management
- Added new components like `Page` and `Container`
- Updated dependencies and added new ones like `jotai` and `lodash`

> The following files were skipped due to too many changes: `packages/localization/src/config/translations.json`, `apps/bridge/utils/atomWithStorageWithErrorCatch.ts`, `apps/bridge/components/GeneralDisclaimerModal/GeneralRiskAcceptModal.tsx`, `apps/bridge/pages/_app.tsx`, `apps/bridge/components/WormHole/WormholeWidget.tsx`, `apps/bridge/components/WormHole/types.ts`, `apps/bridge/components/DisclaimerModal/DisclaimerModal.tsx`, `apps/bridge/components/WormHole/constants.ts`, `apps/bridge/components/WormHole/theme.ts`, `pnpm-lock.yaml`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/bridge/.env.example [hunk 1]
```diff
@@ -0,0 +1,8 @@
+NEXT_PUBLIC_NODE_REAL_API_ETH=
+NEXT_PUBLIC_NODE_REAL_API_GOERLI=
+NEXT_PUBLIC_NODE_REAL_API_ARBITRUM_ONE=
+NEXT_PUBLIC_NODE_REAL_BASE=
+NEXT_PUBLIC_NODE_REAL_SOLANA=
```
Comment (chef-jojo) [inline apps/bridge/.env.example:5]: you can reuse the same key

File: apps/bridge/components/DisclaimerModal.tsx/DisclaimerModal.tsx
Comment (chef-jojo) [inline apps/bridge/components/DisclaimerModal.tsx/DisclaimerModal.tsx:1]: double tsx

File: apps/bridge/package.json [hunk 2]
```diff
@@ -3,27 +3,34 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev -p 3001",
+    "dev": "next dev",
```
Comment (chef-jojo) [inline apps/bridge/package.json:6]: please revert it

**Comment by chefjackson** [comment]
@SocketSecurity ignore npm/@emotion/utils@1.2.1
@SocketSecurity ignore npm/@emotion/cache@11.11.0
@SocketSecurity ignore npm/@emotion/sheet@1.2.2
@SocketSecurity ignore npm/@emotion/weak-memoize@0.3.1
@SocketSecurity ignore npm/@mui/base@5.0.0-beta.36
@SocketSecurity ignore npm/@mui/system@5.15.9
@SocketSecurity ignore npm/@mui/utils@5.15.9
@SocketSecurity ignore npm/@mui/private-theming@5.15.9
@SocketSecurity ignore npm/@mui/styled-engine@5.15.9
@SocketSecurity ignore npm/@mui/material@5.15.10

**Review by chefjackson — COMMENTED** [review]
Please resolve the conflict

**Review by chefjackson — APPROVED** [review]

**Pull request merged**